### PR TITLE
Real time wordsmithing from meeting with business

### PR DESCRIPTION
**Pull request by hogmann**
Bunch of wordsmithing from the meeting with business on the style guide. Mostly focusing around enrollee, but some global fixes.

### CODE REVIEW

**Review by johnnyl6 — APPROVED** [review]
🚀

File: prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.html [hunk 1]
```diff
@@ -286,7 +286,7 @@ <h6 class="mb-0">
                  let-provisioner="provisioner">
       <p class="mb-4">
         Provide the email address of the person in your organization who is responsible for contacting
-        {{ provisioner }} to set up accounts (e.g. your Office Manager or MOA). Use a comma to separate multiple
+        {{ provisioner }} to set up accounts (e.g.,your Office Manager or MOA). Use a comma to separate multiple
```
Comment (AMiKaH) [inline prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.html:289]: Should there be a space between the comma and your?

File: prime-angular-frontend/src/app/modules/paper-enrolment/pages/regulatory-page/regulatory-page.component.html [hunk 2]
```diff
@@ -38,7 +38,7 @@
       <app-page-subheader>
         <ng-container appPageSubheaderTitle>College Licence Information</ng-container>
         <ng-container appPageSubheaderSummary>
-          Provide details for all the enrollee's College licenses. If the enrollee does not have a licence from one of
+          Provide details for all the enrollee's college licenses. If the enrollee does not have a licence from one of
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/paper-enrolment/pages/regulatory-page/regulatory-page.component.html:41]: I think this was missed:  "college licenses" -> "college licences"?

File: prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html [hunk 3]
```diff
@@ -11,11 +11,11 @@
         </ng-container>
         <ng-container #alertContent
                       class="alert-content">
-          <p>You have cancelled your login to PRIME</p>
+          <p>You have cancelled your log in to PRIME</p>
           <p>For assistance with PRIME, contact <app-prime-phone></app-prime-phone> or
             <app-prime-support-email></app-prime-support-email>
           <p>
-            For assistance logging in with the BC Services Card app, contact the Helpdesk
+            For assistance logging in with the BC Services Card app, contact the Help desk
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/components/auth/prime-enrolment-access/prime-enrolment-access.component.html:18]: Should that be "Help Desk"?  The edit in `prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html` indicated that

**Review by AMiKaH — APPROVED** [review]
LGTM

**Review by neophyte57 — APPROVED** [review]
🎇

**Pull request merged**